docs/root/version_history/current.rst

```diff
@@ -78,6 +78,7 @@ Removed Config or Runtime
 New Features
 ------------
 
+* access_log: added the new response flag `OM` for Overload Manager termination. The error flag will be set when the http stream is terminated by overload manager.
```

Comment:
maybe add ref: to the proto, similar to the one below?

Reply:
Done.